google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/GceTestEnvConfig.java

```diff
@@ -53,6 +58,13 @@ public GceTestEnvConfig() {
     if (!serverUrl.isEmpty()) {
       builder.setHost(serverUrl);
     }
+    if (!credentialsFile.isEmpty()) {
```

Comment:
Can you please clarify what this change is?

Reply:
This variable works in the same way as the serverUrl variable. It is read from a custom environment variable named `spanner.gce.config.credentials_file` (defined in `GceTestEnvConfig#GCE_CREDENTIALS_FILE`) and these credentials are used if this custom environment variable has been set. If it has not been set, the integration test will use the default credentials.
The reason that I have added this option is that when I run the integration tests locally, I use a number of different environments that need different credentials, and changing the default credentials of my local environment is a lot more cumbersome than just passing in a custom system property when running the tests.

Reply:
Gotcha - since this is already a larger PR, can we move this out into a separate change?

Reply:
I moved the change to a new PR.